scripts/release_notes/release_notes.ts

```diff
@@ -303,8 +305,19 @@ async function generateNotes(repositories: util.Repo[]) {
   });
 
   // Ask for github token.
-  const token = await util.question(
-      'Enter GitHub token (https://github.com/settings/tokens): ');
```

Comment:
We should still ask for the token if the user hasn't defined it in their `.bashrc`.